src/modules/site-v2/templates/order/order_confirm.html

```diff
@@ -52,7 +52,7 @@ <h3>Order Information</h3>
                 {% for item in items %}
                 <tr>
                     <td>{{ loop.index }}</td>
-                    <td>{{ item['strain'] }}</td>
+                    <td>{{ item['name'] }}</td>
                     <td><em>{{ item['species']|capitalize|replace('_', '. ') }}</em></td>

```

Comment:
Same comment here as above - you can use the species `short_name` property to generate this value